proof/refine/ARM/PageTableDuplicates.thy

```diff
@@ -1220,17 +1225,15 @@ lemma createNewObjects_valid_duplicates'[wp]:
       apply (simp add: word_bits_def)
      apply wp
       apply (wp snoc.hyps)
+      apply (subst conj_assoc[symmetric])
+      apply (subst conj_assoc[symmetric])
+      apply (subst conj_assoc[symmetric])
```

Comment:
Sure,  I can add a comment. I can’t say I have explored all possibilities to avoid this, but I believe what I have is a slight improvement to the previous version. The wp rule used here had three conjuncts and now it has four. And the original proof was stepping it through one by one three times. I didn’t want to make it four times repeats. So, instead, I am bundling up the four conjuncts together first to apply the rule in just one step.